index.html

```diff
@@ -0,0 +1,40 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
+    <link rel="stylesheet" href="style.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Poppins:wght@700&display=swap" rel="stylesheet">
+    <title>Portfolio</title>
+</head>
+<body class="background">
+    <header class="navbar">
+        <div class="logo">
+          <a href="#">MyLogo</a>
+        </div>
+        <div>
+          <a href="#">
+            <i class="fa fa-bars"></i>
+          </a>
+        </div>
+    </header>
+    <section class="onscreen">
+      <h1 class="bold">I'm Amanulla </h1>
+      <h1 class="bold">Glad to see you!</h1>
```

Comment:
Please use only one h1 on the page, this is an HTML good practice to have a professional page.